kolibri/plugins/learn/assets/src/views/LibraryPage/index.vue

```diff
@@ -3,7 +3,6 @@
   <LearnAppBarPage
     :appBarTitle="appBarTitle"
     :appearanceOverrides="{}"
-    :loading="loading"
```

Comment:
**non-blocking:**

along the same lines at @pcenov's [earlier comment](https://github.com/learningequality/kolibri/pull/10455#issuecomment-1544148276) (and hopefully not contrary to his & others' suggestions), i was struck by the variety of different loading indicators, which felt especially evident here in `Learn` - while `Bookmarks` provides the helpful spinny loader, when you navigate to `Library` or `Home`, the loader is the subtler green line loader, and since this was the last part of kolibri i visited during review, the difference felt pretty incongruous, the line loader feels comparatively hard to notice, especially when you're not looking for it, and the reasoning behind having a spinning loader in one subtopic vs the line loader in another subtopic within the same plugin feels a bit hard to understand as a user. even though one may be linked to particular content while the other may be linked to a whole page loading, i think that differentiation isn't obvious (and my understanding may be wrong!) and to a user the effect is the same - they're waiting, but they may not notice the loading indication in all cases.

i don't think this should necessarily be addressed in this PR (or at all if it's a minority opinion), but i did want to make a note of it here. 

https://github.com/learningequality/kolibri/assets/43046460/8f44204b-59b3-4cdb-9b59-70179942db45

Reply:
I agree it's odd -- and looking back at this I think that the [per our KDS docs](https://design-system.learningequality.org/loaders/#linearloaders) I've not applied the correct pattern as this is the "initial page load" - so it should have been the linear loader only (which jives a bit w/ a comment Richard noted about the props in `LearnAppBarPage` relying on the core loading state rather than taking in a more localized prop)

I think that with passing the props up to that page in this case will be a relatively simple update to make, which I can make note of in a follow-up.